packages/lit-dev-tools-cjs/src/api-docs/transformer.ts

```diff
@@ -646,14 +649,6 @@ export class ApiDocsTransformer {
     if (!pos.source) {
       return;
     }
-
-    // TODO(aomarks) The Lit monorepo d.ts.map files currently incorrectly have
-    // a sources field like "../src/source.ts" because they are copied directly
-    // out of the "development/" folder. We need to fix that properly the Lit
-    // monorepo, but temporarily fix it here too.
-    if (pos.source.startsWith('../')) {
-      pos.source = pos.source.slice('../'.length);
-    }
```

Comment:
Hmm... the underlying issue mentioned in this comment actually is not fixed since the `d.ts` files are still being directly copied out of `development/`. I think this is no longer needed because typodoc is now picking up source files from the `development/` directory rather than the copied ones in the package root. In any case, removing this is indeed correct.

Reply:
Thank you for figuring out why removing this worked!